JS/audio.js

```diff
@@ -0,0 +1,37 @@
+function listenKeypress() {
+    document.addEventListener('keypress', function(e) {
+        var keys = {
+            'q': '1',
+            'w': '2',
+            'e': '3',
+            'r': '4'
+        }
+        if (e.key in keys) {
+            var sound = 'cheer' + keys[e.key]
+            makeNoise(sound)
+            var $city = $('#city-' + keys[e.key])
+            shootLaser($city)
+        }
+    })
+}
+
+var cities = document.querySelectorAll('.city')
+cities.forEach(function(city) {
+    city.addEventListener('click', function() {
+        var param = 'cheer' + city.dataset.city
```

Comment:
param is a meaningless variable name, please make it mean something

Reply:
changed in 50aae2a